fbw-a32nx/src/systems/instruments/src/EFB/Ground/Pages/Payload/Seating/Constants.ts

```diff
@@ -1,4 +1,13 @@
-export const TYPE = Object.freeze({ NB_ECO: 0, NB_ECO_EMERG: 1, WB_ECO: 2, WB_ECO_EMERG: 3 });
+export enum TYPE {
+    NB_ECO = 0,
+    NB_ECO_EMERG = 1,
+    WB_ECO = 2,
+    WB_ECO_EMERG = 3,
+    WB_BIZ_FLAT_R = 4,
+    WB_BIZ_FLAT_L = 5,
+    WB_SUITE_R = 6,
+    WB_SUITE_L = 7,
+}
```

Comment:
**suggestion:** If this enum isn't constrained by external dependencies, let's name the members more clearly.

```suggestion
export enum SeatType {
    NarrowbodyEconomy = 0,
    NarrowbodyEconomyEmergencyRow = 1,
    WidebodyEconomy = 2,
    WidebodyEconomyEmergencyRow = 3,
    WidebodyBusinessFlatLeft = 4,
    WidebodyBusinessFlatRight = 5,
    WidebodySuiteRight = 6,
    WidebodySuiteLeft = 7,
}
```